aten/src/ATen/native/cuda/SoftMax.cu

```diff
@@ -1093,7 +1208,11 @@ TORCH_IMPL_FUNC(softmax_cuda_out) (
   const int64_t dim,
   const bool half_to_float,
   const Tensor &output) {
-  host_softmax<SoftMaxForwardEpilogue,false>(input, dim, half_to_float, output);
+#if defined(USE_ROCM) && defined(PYTORCH_USE_FAST_SOFTMAX)
```

Comment:
can we temporarily make this just USE_ROCM so the CI will run with these changes?  putting it behind the PYTORCH_USE_FAST_SOFTMAX means none of this new code will be exercised.

Reply:
Just removed that temporarily.